scripts/docker-compose/check-env.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+#
+# SPDX-FileCopyrightText: 2024 Lifely
+# SPDX-License-Identifier: EUPL-1.2+
+#
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+(
+  cd "$SCRIPT_DIR"
+
+  if sudo lsof -i@localhost:8080; then
+    echo -e "\nPlease stop the currently running WildFly!"
```

Comment:
Can be anything running on port 8080. Does not need to be WildFly/ZAC. I would keep it neutral. Just say that there is process running on this port?